pallets/phala-world/src/incubation.rs

```diff
@@ -98,28 +156,6 @@ pub mod pallet {
 	where
 		T: pallet_uniques::Config<ClassId = CollectionId, InstanceId = NftId>,
 	{
-		/// Once users have received their origin_of_shells and the start incubation event has been
-		/// triggered, they can start the incubation process and a timer will start for the
-		/// origin_of_shell to awaken at a designated time. Origin of Shells can reduce their time
-		/// by being in the top 10 of origin_of_shell's fed per era.
-		///
-		/// Parameters:
-		/// - origin: The origin of the extrinsic starting the incubation process
-		/// - collection_id: The collection id of the Origin of Shell RMRK NFT
-		/// - nft_id: The NFT id of the Origin of Shell RMRK NFT
-		#[pallet::weight(10_000 + T::DbWeight::get().reads_writes(1,1))]
-		#[transactional]
-		pub fn start_incubation(
```

Comment:
I think this is required. Not started nft cannot be fed

Reply:
Since all Origin of Shells have the same hatch time whenever we enable the hatching phase, there wasn't a need for this. The Storage `OriginOfShellFoodStats` will is a `ValueQuery` and holds the total number of time fed each `Era`.

Reply:
Added this back in. Now Users cannot feed Origin of Shells unless they have started the Incubation process.